README.md

```diff
@@ -1,23 +1,30 @@
-GDB Scripts
------------
+# GDB Scripts
+
+## Audience
 
-* `Audience:`
    This section is written for those with entry level understanding of GDB.
 
-* `Introduction:`
+## Introduction
+
    This repo contains gdb scripts to peruse memory of restraintd.  It is
    especially useful to derive data from memory as a result of a crash.
 
-*  `Steps if restraintd crashed:`
+## Steps if restraintd crashed
+
    If restraintd crashes, you gather the core file and prepare an
    environment similar to that which restraint crashed. So if the crash
    occured in RHEL7, then you install a machine with RHEL7 and same
    version of restraintd. You also need the following installed:
 
-      `yum install gdb`
+       ```
+       yum install gdb
+       ```
 
-   Then do
-      `gdb restraintd the-core-file-found-from-crash`
+  Then do
+
+    ```
+    gdb restraintd the-core-file-found-from-crash
```

Comment:
Alignment in this line is different from the other command-like lines. Is that intentional?

Reply:
yeah was trying to get ``` to work. It's still not working but it's not too important.  These are tools for us to use.  The biggest purpose of this change was for me to practice the git workflow Martin showed me this morning.